Makefile

```diff
@@ -12,7 +12,7 @@ FILE_SO= libsregex.so
 
 CC= gcc
 CFLAGS+= -fpic -g -Wall -O3 -Isrc -I.
-PREFIX= /usr/local
+PREFIX?= /usr/local
```

Comment:
I think we can just override PREFIX from the command-line like below?

```bash
make PREFIX=/path/to/my/prefix
```

This will override any internal assignments to the `PREFIX` variable in Makefile even without using `?=`.